Package.swift

```diff
@@ -30,10 +30,10 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-atomics.git", from: "1.1.0"),
-        .package(url: "https://github.com/apple/swift-crypto.git", from: "1.0.0"),
+        .package(url: "https://github.com/apple/swift-crypto.git", from: "2.0.0"),
```

Comment:
This can be 1.0.0 to 3.0.0

Reply:
Yes this was the recommendation. Swift-metrics had to make a breaking change to fix something. But this meant the major version being used was quite varied across the eco-system. So the suggestion was to allow version 1.0.0 ..< 3.00.

Reply:
It can, but shouldn't. Anything that's still referring to SwiftCrypto 1.x without allowing the 2.x range is now so badly out of date that it _should_ be forced to update; being compatible with such out of date dependencies is a bad idea.

Reply:
We could do the same for swift-crypto as @0xTim suggests. I don't require v2.0 so why should I force it on my users.

Reply:
Not to mention that given the 5.6 minimum Swift requirement, this package effectively requires SwiftCrypto >= 2.5.0

Reply:
The recommendation from Cory when he released 2.0.0 was to support the 1.0.0 ..< 3.0.0 range. Given Soto only uses MD5, SHA1 and SHA256 don't think requiring 2.0.0 is necessary.